crates/scroll/cli/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "reth-scroll-cli"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+# reth
+reth-db = { workspace = true, features = ["scroll"] }
+reth-cli.workspace = true
+reth-cli-commands.workspace = true
+reth-cli-runner.workspace = true
+reth-node-builder.workspace = true
+reth-node-core.workspace = true
+reth-node-metrics.workspace = true
+reth-tracing.workspace = true
+
+# scroll
+reth-scroll-evm.workspace = true
+reth-scroll-node.workspace = true
+
+# misc
+eyre.workspace = true
+clap.workspace = true
+tracing.workspace = true
+
+[features]
+optimism = []
```

Comment:
nit: do we need this feature?

Reply:
Unfortunately yes, mostly because of `reth-primitives` and `revm` which need to be compiled with `#[cfg(all(feature = "scroll", not(feature = "optimism")))]` in order to correctly feature gate conflicting fields or types.